### ISSUE TITLE
[Player and Coach styles' clustering using UMAP and GMM]

### ISSUE THREAD

**Issue by yashkarle** [yashkarle/data-science-notes#3]
## TL;DR
List of ideas, articles and code snippets to understand ...

### Blog Posts
https://theathletic.com/3044091/2022/01/02/the-five-kingdoms-of-football/
https://theathletic.com/3050603/2022/01/05/who-never-fouls-who-presses-who-hoofs-it-what-our-data-experiment-you-about-your-team/

### Project Idea
Very simply, the idea is to rely on John Muller's analysis done for teams and extend it for players and coaches

## Key theoretical takeaways
* UMAP - Uniform Manifold Approximation and Projection for dimension reduction
https://umap-learn.readthedocs.io/en/latest/

* GMM - Gaussian Mixture Models
https://scikit-learn.org/stable/modules/mixture.html
